docset/winserver2012-ps/failoverclusters/Test-Cluster.md

```diff
@@ -216,6 +152,7 @@ Accept wildcard characters: False
 ```
 
 ### -Force
+
 Runs the cmdlet without prompting for confirmation.
 By default the cmdlet will ask for confirmation from the user before proceeding.
```

Comment:
Elsewhere in the documentation it calls out that the **Force** parameter takes the disks offline for the duration of the tests if needed. I think that's worth pointing out here, too.

Reply:
Good spot, I've updated this in all versions.